ActionMenu/ActionMenu.cs

```diff
@@ -257,6 +257,7 @@ public Menus BuildMelonPrefsMenus(List<MelonPreferences_Entry> melonPrefs)
             "index.html", "index.js", "index.css", "actionmenu.json",
             "icon_actionmenu.svg", "icon_menu.svg", "icon_back.svg", "icon_avatar_emotes.svg", "icon_melon.svg",
             "icon_avatar_settings_profile.svg",
+            "icon_media_fwd.svg", "icon_media_prev.svg", "icon_media_pause.svg", "icon_media_play.svg",
```

Comment:
(1) There is no need to add them to the project, they're already available, check `ChilloutVR_Data\StreamingAssets\Cohtml\UIResources\CVRTest\gfx\btn-media-next.svg`  (2) CVR assets is under copyright, we cannot add them to a GPL / free repository.